src/test/java/de/telran/processor/services/ActionsConfigServiceTest.java

```diff
@@ -0,0 +1,27 @@
+package de.telran.processor.services;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class ActionsConfigServiceTest {
+    ActionsConfigService service = new ActionsConfigService();
+
+    public ActionsConfigServiceTest() throws IOException {
+    }
+
+    @org.junit.Test
+    public void getActionPackage() {
+        String answer = service.getActionPackage();
+        assertEquals("de.telran.processor.action", answer);
+    }
+
+    @org.junit.Test
```

Comment:
You should import `org.junit.Test` in order to omit the package prefixing in the annotation.